tcmur_device.c

```diff
@@ -437,7 +437,7 @@ int tcmu_acquire_dev_lock(struct tcmu_device *dev, uint16_t tag)
 	else
 		rdev->lock_state = TCMUR_DEV_LOCK_UNLOCKED;
 
-	tcmu_dev_dbg(dev, "lock call done. lock state %d\n", rdev->lock_state);
+	tcmu_dev_info(dev, "lock call done. lock state %d\n", rdev->lock_state);
```

Comment:
If we are going to make it a "info" level message then let's make it more user friendly.  Something like:

tcmu_dev_info(dev, "Lock acquisition %s\n", rdev->lock ==  TCMUR_DEV_LOCK_LOCKED ? "successful" : "unsuccessful");

Reply:
@mikechristie 
Done.
Thanks.